### ISSUE TITLE
glmmTMB error

### ISSUE THREAD

**Issue by ivschroj** [lionel68/addSE#3]
Hi Lionel,
The add_se function gives the following error for me: 
Error in data.frame(Coef = linkinv(coef_f), LCI = linkinv(coef_f - 1.96 * : could not find function "linkinv"

The model I try to implement is the following:
glmmTMB(Dabun ~ fragscale * speccomb + (1|id_plot) + (1|period), data = div_qrob,
                family=nbinom2(link="log"), offset=circ)
Dabun is count data, fragscale a continuous and speccomb a factorial variable.

**Comment by lionel68** [comment]
Thanks for reporting this.

I cannot reproduce your error with some example data, could you try this:

```
data("Salamanders");
m2 <- glmmTMB(count~spp + mined + (1|site), family=nbinom2(link="log"), Salamanders);
add_se(m2, "spp")
```

And report what you get?

**Comment by ivschroj** [comment]
I get the same error again:
Error in data.frame(Coef = linkinv(coef_f), LCI = linkinv(coef_f - 1.96 * :
could not find function "linkinv"

Even after making sure I have the latest version of add_se.

2018-08-16 15:22 GMT+02:00 lionel68 <notifications@github.com>:

**Comment by lionel68** [comment]
Hmm ... Do you have the last version of glmmTMB installed?

Try running the following code:

``
if(packageVersion("glmmTMB") != "0.2.2.0") install.packages("glmmTMB")
``

**Comment by ivschroj** [comment]
running that code made my R crash (for the umpteenth time last days....).
I did install glmmTMB again, and my version is 0.2.0. So apparently not the
latest version.
However, I am still getting the same error.
Any other packages that could be the problem? These are the packages I load
for my script:

# Libraries (possibly) needed
 # Definitely
library(dplyr)
library(tidyr)
library(addSE)
library(MASS)
library(vegan)
library(ggplot2)
library(lme4)
library(lmerTest)
library(mvabund)
library(glmmTMB)
library(reshape)

# Maybe

library(lazyeval)
library(permute)
library(labeling)
library(ggthemes)
library(grid)
library(gridExtra)
library(digest)
library(rda)
library(AICcmodavg)
library(cowplot)

2018-08-17 13:20 GMT+02:00 lionel68 <notifications@github.com>:

**Comment by ivschroj** [comment]
Maybe this helps. This is what it says when loading the glmmTMB. I guess
updating my Matrix package may help?
package �glmmTMB� was built under R version 3.3.3Package version
inconsistency detected.
TMB was built with Matrix version 1.2.14
Current Matrix version is 1.2.8
Please re-install 'TMB' from source using install.packages('TMB', type =
'source') or ask CRAN for a binary version of 'TMB' matching CRAN's
'Matrix' packageWarning messages:
1: In readLines(input, encoding = "UTF-8") :
  incomplete final line found on 'C:/Program
Files/R/R-3.3.2/library/Gmisc/rmarkdown/templates/CONSORT_v2010_docx/template.yaml'
2: In readLines(input, encoding = "UTF-8") :
  incomplete final line found on 'C:/Program
Files/R/R-3.3.2/library/Gmisc/rmarkdown/templates/STROBE_v4_docx/template.yaml'
3: In readLines(input, encoding = "UTF-8") :
  incomplete final line found on 'C:/Program
Files/R/R-3.3.2/library/raw/rmarkdown/templates/reserveReview/template.yaml'

2018-08-17 14:42 GMT+02:00 Irene Lantman <irene.lantman@gmail.com>:

**Comment by ivschroj** [comment]
Issue was solved via updating glmmTMB to version 0.2.2.0
